account_payment_order_validation_move_line/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds the possibility to choose only validated invoice for the account.payment.line.create and it can be set from theaccount.payment.mode model and account.payment.line.create
```

Comment:
The module adds the possibility to filter only invoices that have been marked as "To pay" on the payment order.

Or something like that (do not speak about technical models here as any functional person won't be able to understand)